src/components/EventCard/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Link } from "gatsby";
```

Comment:
Remove the `Link` since it is not used anywhere

Reply:
removed